### PR TITLE
Supplement Test Cases In XmlBeanFactoryTests

### PR DESCRIPTION
**Pull request by lixiaolong11000**
Supplement Test Cases In XmlBeanFactoryTests

### CODE REVIEW

File: spring-context/src/test/resources/org/springframework/beans/factory/xml/XmlBeanFactoryTests-reftypes.xml [hunk 1]
```diff
@@ -14,6 +14,28 @@
 		<property name="spouse" ref="jenny"/>
 	</bean>
 
+	<bean id="jennyc"  class="org.springframework.tests.sample.beans.TestBean">
+		<constructor-arg index="1"><ref bean="davidc"/></constructor-arg>
+	</bean>
+
```
Comment (sbrannen) [inline spring-context/src/test/resources/org/springframework/beans/factory/xml/XmlBeanFactoryTests-reftypes.xml]: I'm guessing `c` stands for constructor and `d` stands for depends-on, but what does `s` stand for?

In any case, I'd recommend more meaningful names -- for example, perhaps with suffixes such as `_constructor`, `_depends_on`, and `_prototype`.

**Review by sbrannen — CHANGES_REQUESTED** [review]
I've requested a few minor changes.

**Comment by lixiaolong11000** [comment]
ok, it sounds good!  I fixed it.

File: spring-context/src/test/java/org/springframework/beans/factory/xml/XmlBeanFactoryTests.java [hunk 2]
```diff
@@ -536,6 +536,51 @@ public void testCircularReferences() {
...
+		reader.loadBeanDefinitions(REFTYPES_CONTEXT);
+		xbf.getBean("jennys");
+		assertThatExceptionOfType(BeanCreationException.class).isThrownBy(() ->
+				xbf.getBean("jennys"))
+				.matches(ex -> ex.contains(BeanCurrentlyInCreationException.class));
+		assertThatExceptionOfType(BeanCreationException.class).isThrownBy(() ->
+				xbf.getBean("davids"))
+				.matches(ex -> ex.contains(BeanCurrentlyInCreationException.class));
+	}
+
```
[the remark this answers is not in the document]
Comment (lixiaolong11000) [reply spring-context/src/test/java/org/springframework/beans/factory/xml/XmlBeanFactoryTests.java]: I removed the next line.

**Comment by lixiaolong11000** [comment]
@sbrannen  hi,I had fixed all the problems above

**Review by sbrannen — APPROVED** [review]

**Pull request closed without merging**